eq-author/src/components/Header/SharingModal/UserSearch.js

```diff
@@ -110,7 +110,7 @@ const UserSearch = ({ users, onUserSelect }) => {
                       }
                       const value = inputValue.toLowerCase();
                       return (
-                        user.name.toLowerCase().includes(value) ||
+                        (user.name || "").toLowerCase().includes(value) ||
                         user.email.toLowerCase().includes(value)
```

Comment:
Could we do `(user.name || user.email).toLowerCase().includes(value);`?

Reply:
We want to search both